cmd/test-utils_test.go

```diff
@@ -2361,3 +2361,24 @@ func getEndpointsLocalAddr(endpoints EndpointList) string {
 
 	return globalMinioHost + ":" + globalMinioPort
 }
+
+// fetches a random number between range min-maz
+func getRandomRange(min, max int) int {
+	rand.Seed(time.Now().Unix())
+	return rand.Intn(max-min) + min
+}
+
+// randomizes the order of bytes in the byte array.
```

Comment:
Add a comment attributing the algorithm "Knuth Fisher-Yates shuffle" - so its clear where it comes from.